puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/3.10.groovy

```diff
@@ -39,42 +39,14 @@ pipeline {
         stage('Install Test') {
             parallel {
                 stage('Install test') {
-                    agent { label 'el' }
-
                     steps {
-
-                        git_clone_foreman_infra()
-
-                        withCredentials([string(credentialsId: 'centos-jenkins', variable: 'PASSWORD')]) {
-                            runPlaybook(
-                                playbook: 'ci/centos.org/ansible/jenkins_job.yml',
-                                inventory: 'localhost',
-                                extraVars: [
-                                  "jenkins_job_name": "foreman-katello-3.10-test",
-                                  "jenkins_username": "foreman"
-                                ],
-                                sensitiveExtraVars: ["jenkins_password": "${env.PASSWORD}"]
-                            )
-                        }
+                        runCicoJob("foreman-katello-3.10-test")
                     }
                 }
                 stage('Upgrade test') {
-                    agent { label 'el' }
                     steps {
-
-                        git_clone_foreman_infra()
                         sleep(5) //See https://bugs.centos.org/view.php?id=14920
```

Comment:
This may break because there's no agent to run on. Because of the duplication I'm going to move it inside the function.